src/components/companion/blueprintProfiles/attendeeInfo.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { AnimatedBorder } from '../../ui/animated-border';
+
+interface UserProfile {
+    name: string;
+    role: string;
```

Comment:
What do we expect role to say, 'Attendee'?

Reply:
yea, I think we expect role to be either: "attendee", "delegate", or "company" - this can be changed though if needed.